src/main/java/bit/javaoop/Company.java

```diff
@@ -0,0 +1,42 @@
+package bit.javaoop;
+
+public class Company {
+    private String companyName;
+    private CEO ceo;
+    private static Company instance = new Company("Java OOP");
```

Comment:
Singleton to antypattern! Postaraj się zapomnieć o jego istnieniu i jedyne co powinno Ci się z nim kojarzyć to to żeby go nie używać.